### PR TITLE
 cancel booking new design added

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?
UI change for /cancel/:id page

TODO

- [x]  make changes in common.json
- [x]  change recurring meeting cancel ui 

Fixes # (issue)  #3554 
Screenshot 2022-08-03 at 2 37 48 AM

### CODE REVIEW

**Review by sean-brydon — COMMENTED** [review]
Minor NIT :)

File: apps/web/pages/cancel/[uid].tsx [hunk 1]
```diff
@@ -81,10 +83,10 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
                       </div>
                       <div className="mt-3 sm:mt-5">
                         <h3
-                          className="text-center text-lg font-medium leading-6 text-gray-900"
+                          className="text-center text-lg text-[24px] font-bold leading-6 text-gray-900"
```
[the remark this answers is not in the document]
Comment (Udit-takkar) [reply apps/web/pages/cancel/[uid].tsx]: done

**Comment by joeauyeung** [comment]
@Udit-takkar I approved deploying a preview on Vercel and things look good. Just need to solve the conflicts and then we can approve

**Comment by Udit-takkar** [comment]
cc @CarinaWolli

File: apps/web/pages/cancel/[uid].tsx [hunk 2]
```diff
@@ -98,29 +100,30 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
-                                </p>
-                              </div>
-                            )}
-                          <div className="text-gray-500">
-                            <div className="flex flex-row items-start justify-center space-x-3">
+                        <div className="ptext-left mt-8 grid grid-cols-3 border-t border-b py-8">
+                          <h2 className="font-medium">{t("event_name")}</h2>
+                          <h2 className="col-span-2 mb-12">{props.booking?.title}</h2>
+
+                          <h2 className="font-medium">{t("when")}</h2>
```
Comment (zomars) [inline apps/web/pages/cancel/[uid].tsx:107]: I don't think having 3 consecutive `h2` tags would be semantically correct.

File: apps/web/pages/cancel/[uid].tsx [hunk 3]
```diff
@@ -98,29 +100,30 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
+                                      recurringCount: props.recurringInstances.length,
+                                    })}
+                                  </p>
+                                </div>
+                              )}
+                            <div className="col-span-2 flex flex-row items-start ">
                               {props.booking?.eventType.recurringEvent && props.recurringInstances ? (
                                 <>
-                                  <Icon.Calendar className="mt-2 ml-1 h-4 w-4" />
+                                  {/* <Icon.Calendar className="mt-2 ml-1 h-4 w-4" /> */}
```
Comment (zomars) [inline apps/web/pages/cancel/[uid].tsx:126]:
```suggestion
```

Remove if unused

File: apps/web/pages/cancel/[uid].tsx [hunk 4]
```diff
@@ -152,28 +155,35 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
+                                </div>
                               )}
                             </div>
                           </div>
                         </div>
                       </div>
                     </div>
                     {props.cancellationAllowed && (
-                      <div className="mt-5 sm:mt-6">
+                      <div className="mt-5 sm:mt-6  lg:space-y-2">
```
Comment (zomars) [inline apps/web/pages/cancel/[uid].tsx:178]:
```suggestion
                      <div className="mt-5 sm:mt-6 lg:space-y-2">
```

**Review by zomars — CHANGES_REQUESTED** [review]

File: apps/web/pages/cancel/[uid].tsx (hunk 2 above)
Comment (Udit-takkar) [reply apps/web/pages/cancel/[uid].tsx:107]: ok what should i use then?

File: apps/web/pages/cancel/[uid].tsx [hunk 5]
```diff
@@ -98,29 +100,30 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
-                                    t,
-                                    recurringEvent: props.booking.eventType.recurringEvent,
-                                    recurringCount: props.recurringInstances.length,
-                                  })}
-                                </p>
-                              </div>
-                            )}
-                          <div className="text-gray-500">
-                            <div className="flex flex-row items-start justify-center space-x-3">
+                        <div className="ptext-left mt-8 grid grid-cols-3 border-t border-b py-8">
```
Comment (zomars) [inline apps/web/pages/cancel/[uid].tsx:103]: Is `ptext-left` a valid class name?

File: apps/web/pages/cancel/[uid].tsx (hunk 2 above)
Comment (zomars) [reply apps/web/pages/cancel/[uid].tsx:107]: Leave it as it was please 🙏🏽 the original issue only mentioned button alignment

File: apps/web/pages/cancel/[uid].tsx (hunk 5 above)
Comment (Udit-takkar) [reply apps/web/pages/cancel/[uid].tsx:103]: no typo

File: apps/web/pages/cancel/[uid].tsx (hunk 2 above)
Comment (Udit-takkar) [reply apps/web/pages/cancel/[uid].tsx]: replaced it with div like it is done on /success page.
Original issue was about button alligment but new design was to be used for this page.

**Comment by Udit-takkar** [comment]
rebase and resolved conflict @zomars

**Pull request merged**